core/crypto/aes-cts.c

```diff
@@ -220,6 +220,7 @@ static void cts_copy_state(struct crypto_cipher_ctx *dst_ctx,
 
 	crypto_cipher_copy_state(dst->cbc, src->cbc);
 	crypto_cipher_copy_state(dst->ecb, src->ecb);
+	dst->mode = src->mode;
```

Comment:
The GP spec says:
--------
### TEE_CopyOperation()
**Panic Reasons**
* If the algorithm or mode differ in `dstOperation` and `srcOperation`"
--------

...so it seems we should check the values rather than blindly copying it?

Reply:
Yeah, but we're only setting with with `cts_init()`, which isn't called in this case. I suppose we could add a check in `TEE_CopyOperation()`.

Reply:
> Yeah, but we're only setting with with `cts_init()`, which isn't called in this case.

Ah, OK so this is correct.

> I suppose we could add a check in `TEE_CopyOperation()`.

Yep.